srtcore/core.cpp

```diff
@@ -9724,7 +9724,7 @@ int srt::CUDT::checkLazySpawnTsbPdThread()
         // Take the last 2 ciphers from the socket ID.
         tns1 << m_SocketID;
```

Comment:
I would rather do:
```
tns1 << setfill('0') << setw(2) << m_SocketID;
```

The idea here is to use the last two ciphers of the socket number, so for value 2 it would be "02".

Reply:
No problem, that's good.